### ISSUE TITLE
Swap the order of Previews and Pages sections

### ISSUE THREAD

**Issue by cody-clio** [ViewComponent/lookbook#546]
### Describe the solution you'd like

Request the ability to configure the order in which Previews and Pages are rendered in the side panel. We want to be able to focus our users towards the pages rather than the previews, given the type of content we provide within a page. 

### Additional context

![Screenshot 2023-11-07 at 3 07 40 PM](https://github.com/ViewComponent/lookbook/assets/120416026/f2b8199f-b03f-4af2-af53-41cb7c3b1c3e)

**Comment by allmarkedup** [comment]
Hey @cody-clio, really sorry for the slow reply.

This would be a great feature. I've actually thought about it myself (along with being able to rename those sections) many times but never quite gotten around to implementing it. I'll try to use this reminder as the impetus I need to work on it as soon as I have a chance.

**Comment by allmarkedup** [comment]
@cody-clio I've just opened PR #553 that adds a config option to control the order of panels in the sidebar. If you have a chance could you give it a try and let me know if you run into any problems? 

```sh
gem "lookbook", github: "ViewComponent/lookbook", branch: "order-sidebar-nav-sections"
```

**Comment by allmarkedup** [comment]
Released in v2.2.0.
